src/status_im/subs.cljs

```diff
@@ -482,6 +482,12 @@
  (fn [[macc acc]]
    (some #(when (= (:address %) (:address acc)) %) (:accounts macc))))
 
+(re-frame/reg-sub
+ :multiaccounts-count
```

Comment:
maybe it could be a predicate `multiple-multiaccounts?` here since that is the only usage of this sub

Reply:
ok, sounds reasonable

Reply:
done